src/components/home/ProjectCard.tsx

```diff
@@ -0,0 +1,35 @@
+import Image, { StaticImageData } from "next/image";
+import Link from "next/link";
+
+interface ProjectCardProps {
+  project: string;
+  image: StaticImageData;
+  link: string;
+}
+
+const ProjectCard = ({ project, image, link }: ProjectCardProps) => {
+  return (
+    <Link
+      href={link}
+      className="relative my-5 flex flex-row items-center overflow-hidden rounded-lg p-4"
```

Comment:
items-center does nothing here, flex-row is by default applied when you do flex
```suggestion
      className="relative my-5 flex overflow-hidden rounded-lg p-4"
```